### PR TITLE
[job status change notif] parse job config to DB

### PR DESCRIPTION
**Pull request by suiguoxin**
[Work Items](https://github.com/microsoft/pai/issues/5337#issuecomment-839551679)
- DB Framework table definition
- save framework config to DB when submit jobs
- dbc lint

### CODE REVIEW

File: src/database-controller/src/common/framework.js [hunk 1]
```diff
@@ -188,6 +189,12 @@ class Snapshot {
       'extras.hivedscheduler.jobPriorityClass',
       null,
     );
+    // Job status change notification
+    const jobStatusChangeNotification = _.get(
+      loadedConfig,
+      'extras.hivedscheduler.jobStatusChangeNotification',
+      null,
```
Comment (hzy46) [inline src/database-controller/src/common/framework.js:196]: please change this to null

File: src/database-controller/src/common/framework.js [hunk 2]
```diff
@@ -204,6 +211,22 @@ class Snapshot {
       ),
       totalTaskRoleNumber: this._snapshot.spec.taskRoles.length,
       logPathInfix: this._snapshot.metadata.annotations.logPathInfix,
+      notificationAtRunning: jobStatusChangeNotification.running
```
Comment (hzy46) [inline src/database-controller/src/common/framework.js:214]: `_.get(jobStatusChangeNotification, 'running', false)`

File: src/database-controller/src/common/framework.js (hunk 1 above)
Comment (suiguoxin) [reply src/database-controller/src/common/framework.js]: > please change this to `{}`

Fixed, thx

File: src/database-controller/src/common/framework.js (hunk 2 above)
Comment (suiguoxin) [reply src/database-controller/src/common/framework.js]: > `_.get(jobStatusChangeNotification, 'running', false)`

Fixed, thx

**Review by hzy46 — APPROVED** [review]

**Pull request merged**